crates/uv/src/commands/pip_show.rs

```diff
@@ -120,21 +157,36 @@ pub(crate) fn pip_show(
                 .simplified_display()
         )?;
 
+        if let Some(url) = distribution.as_editable() {
+            let path = url.to_file_path().unwrap().simplified_display().to_string();
```

Comment:
Can we avoid this `unwrap`?

Reply:
Do you need to call `to_string`? Isn't display going to be called in `writeln!`?

Reply:
I think so!